dali/test/python/test_fw_iterators.py

```diff
@@ -441,6 +448,7 @@ def check_iterator_results(
     return (ids, sample_counter, per_gpu_counter, epoch_counter, rounded_shard_size)
 
 
+@attr("mxnet")
 def check_mxnet_iterator_pass_reader_name(
```

Comment:
`yield check_mxnet_iterator_pass_reader_name, ...` caused nose2 to lose the attribute - it has to be added to this function, too - unlike for nosetests, which require it on the generator.